src/enclave/outside/attestation/generateQuote.cpp

```diff
@@ -0,0 +1,91 @@
+#include <enclave/outside/attestation/attestation.h>
```

Comment:
Unfortunately, this method can only run in `Hardware` mode, which means it is basically untested.

Reply:
I just noticed these file names and the header structure. We only use Pascal case in file names when they hold a class. We also usually have `.h` and `.cpp` files with the same names where possible, as it makes the code easier to navigate. For functions like this we'd put them in a file together with a snake case name. In this case I think you could combine `generateQuote.cpp` and `validateQuote.cpp` into `attestation.cpp` as they're defined in `attestation.h`. Is there a reason they're spread out?